hyperon_das/das.py

```diff
@@ -386,9 +386,7 @@ def get_link_handles(self, link_filter: LinkFilter) -> List[str]:
         """
         return self.query_engine.get_link_handles(link_filter)
 
-    def get_incoming_links(
-        self, atom_handle: str, **kwargs
-    ) -> tuple[int | None, IncomingLinksT | Iterator]:
+    def get_incoming_links(self, atom_handle: str, **kwargs) -> IncomingLinksT | Iterator:
```

Comment:
I think the `Keyword Args` should be revised. For example. If I pass `no_iterator=False` an exception is raised. And the `cursor` is no longer an available parameter.

Reply:
Fixed the docstring.

The mentioned exception is because the changed code need to run with the new version of AtomDB (0.8.3 already in place). So no fix is required.

As for `cursor` not being used, we don't perform check for unused parameters (e.g. user can pass `blahblah` as a kwarg and it'll work) so I don't see the reason to make a check for `cursor`. Anyway, as mentioned above, I fixed the docstring.